src/analyses/custom_bitvector_analysis.cpp

```diff
@@ -536,7 +536,20 @@ void custom_bitvector_domaint::transform(
     }
     break;
 
-  default:
+  case CATCH:
+  case THROW:
+  case RETURN:
+  case ATOMIC_BEGIN:
+  case ATOMIC_END:
+  case END_FUNCTION:
+  case LOCATION:
+  case START_THREAD:
+  case END_THREAD:
+  case SKIP:
+  case ASSERT:
+  case ASSUME:
+  case INCOMPLETE_GOTO:
+  case NO_INSTRUCTION_TYPE:
     {
     }
```

Comment:
I'd prefer to have a `break` here to avoid stupid bugs when someone adds a new case at the end (several occurrences).